### ISSUE TITLE
Regression: extra floating point casts

### ISSUE THREAD

**Issue by rfalke** [uxmal/reko#897]
Subject: https://github.com/rfalke/decompiler-subjects/blob/master/from_holdec/simple_float_ops/ia32_elf_clang_O0/

Before:
```
void use(real64 rArg04)
{
     printf("%f", rArg04);
}
```

Now:
```
void use(real64 rArg04)
{
	printf("%f", (real64) (uint128) (uint64) (real64) (uint128) (uint64) rArg04);
}
```

**Comment by uxmal** [comment]
Once the `convert` branch is merged into `master`, this issue will be addressed. It's another case of conflation of zero-extensions vs real data type conversions.

**Comment by uxmal** [comment]
As of commit c20ee77f4 this is no longer an issue.
